src/core/processing/models/qgsprocessingmodelalgorithm.cpp

```diff
@@ -424,7 +434,7 @@ QStringList QgsProcessingModelAlgorithm::asPythonCode( const QgsProcessing::Pyth
           importLines << importString;
       }
 
-      lines << QStringLiteral( "import processing" );
+      lines << QStringLiteral( "from qgis import processing" );
```

Comment:
I wasn't planning on changing this, because it causes breakage with older qgis versions which we can easily avoid

Reply:
Yes, I understand. I was doing what has been done in the script template:
https://github.com/qgis/QGIS/blob/master/python/plugins/processing/script/ScriptTemplate.py#L21

Reply:
Hm, we should probably revisit that...